src/index.ts

```diff
@@ -218,42 +208,48 @@ export const createWapeSaleInstance = (
     return mintlist;
   };
 
-  const instance: WapeSaleInstance = {
-    getSalePrice: async (): Promise<string> => {
-      const contract = await getWapeSaleContract(
+  const instance: SaleInstance = {
+    getSalePrice: async (): Promise<PriceInfo> => {
+      const contract = await getSaleContract(
         config.web3Provider,
         config.contractAddress
       );
-      const price = await contract.salePrice();
-      return ethers.utils.formatEther(price).toString();
+      const saleConfig = await contract.saleConfiguration();
```

Comment:
Still in thinking, most of `get` functions are calling `saleConfiguration`, it might delay a bit more time to get result. 
What do you think @RyanCross

Reply:
Truthfully the cost is the network call, and it's a network call either way to get the most up to date info.  The impact of some extra bytes is insignificant.

Or were you getting at something else?

Reply:
maybe can test and see the difference with or without it. unless proven to be much slower, I would agree with Ryan and pull it from chain to get the most accurate value.